pom.xml

```diff
@@ -7,6 +7,20 @@
     <groupId>com.aquaq.training</groupId>
     <artifactId>Java-Refresher-Practicals</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>4.8.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
```

Comment:
Let's go with the core package for JUnit 5 and latest version 5.9.0 https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-engine/5.9.0